src/footer/styles.jsx

```diff
@@ -0,0 +1,10 @@
+import { makeStyles } from '@material-ui/core'
+
+export const useStyles = makeStyles({
+  paragraph: { color: 'white' },
+  footer: { backgroundColor: 'blue', height: '10%' },
+  gridHeight: { height: '100%' },
```

Comment:
naming classes as gridHeight or divHeight is prone to failure of sending intent. this class might add in the future other styling properties 
what about naming it as title? 
and divPageHeight to homePage?

Reply:
Done